valispace/__init__.py

```diff
@@ -306,7 +333,15 @@ def get_component_list(self, workspace_id=None, workspace_name=None, project_id=
 		elif tag_name:
 			url = self.__increment_url(url) + "tags__name={}".format(tag_name)
 
-		return requests.get(url, headers=self.get_request_headers()).json()
+		response = requests.get(url, headers=self.get_request_headers())
+
+		if response.status_code != 200:
```

Comment:
Are these prints for debug or should it stay in the code?

Reply:
These prints can be left here, I think.